src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -1,30 +1,76 @@
 import { Registration, RegistrationStatus } from '../../types/registration.types';
-import { Box, Typography } from '@mui/material';
+import { Box, Link, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
+import { getTitleString } from '../../utils/registration-helpers';
+import { useQuery } from '@tanstack/react-query';
+import { fetchRegistration } from '../../api/registrationApi';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { PageSpinner } from '../../components/PageSpinner';
 
 interface DeletePublicationInformationProps {
   registration: Registration;
 }
 
+function extractOriginalIdentifier(registration: Registration): string | null {
+  return registration.duplicateOf ? getIdentifierFromId(registration.duplicateOf) : null;
+}
+
 export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
   const { t } = useTranslation();
+
+  const originalIdentifier = extractOriginalIdentifier(registration);
+
+  const originalRegistrationQuery = useQuery({
+    queryKey: ['registration', originalIdentifier],
+    queryFn: () => fetchRegistration(originalIdentifier ?? ''),
+    enabled: !!originalIdentifier,
+  });
+  const originalRegistration = originalRegistrationQuery.data;
+
+  const originalRegistrationMainTitle = getTitleString(originalRegistration?.entityDescription?.mainTitle);
+
   return (
     <>
       {(registration.status === RegistrationStatus.Deleted ||
         registration.status === RegistrationStatus.Unpublished) && (
-        <Box
-          sx={{
-            display: 'flex',
-            justifyContent: 'center',
-            backgroundColor: 'registration.main',
-            padding: '1rem',
-            gap: '1.5rem',
-            my: '1rem',
-          }}>
-          <Typography variant="h2" component="h1">
-            {t('registration.result_is_deleted')}
-          </Typography>
-        </Box>
+        <>
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'center',
+              backgroundColor: 'registration.main',
+              padding: '1rem',
+              gap: '1.5rem',
+              my: '1rem',
+            }}>
+            <Typography variant="h2" component="h1">
+              {t('registration.result_is_deleted')}
+            </Typography>
+          </Box>
+
+          {registration.duplicateOf && (
+            <Box
+              aria-live="polite"
+              aria-busy={originalRegistrationQuery.isLoading}
+              sx={{
+                display: 'flex',
+                justifyContent: 'center',
+                backgroundColor: 'registration.main',
+                padding: '1rem',
+                gap: '1.5rem',
+                my: '1rem',
+              }}>
+              {originalRegistrationQuery.isLoading ? (
+                <PageSpinner />
```

Comment:
PageSpinner skal ha en `aria-label` eller `aria-labelledby` som sier noe om hva som lastes. Usikker på hva som blir mest riktig her da... Kanskje `aria-lable={t('registration.citation_points_to')}` elns 🤔 

For å fremprovosere sånne ting bruker jeg personlig `axe devtools`, så setter jeg noe sånt for at spinner skal vises hele tiden: `originalRegistrationQuery.isLoading || !originalRegistrationQuery.isLoading ?<PageSpinner /> : (...)`. Da kan du kjøre en axe test fra nettleseren så får man beskjed om hva som er feil. Eks:

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/28c4e8f4-2845-4458-842e-a80a3f519d5a)

Reply:
Trodde aria-busy skulle fikse saken. Må lese meg opp ser det ut til.

Reply:
Bare legg til relativt logisk `aria-label` eller `aria-labelledby` på `<PageSpinner>` så er denne good 👍